A1constants.py

```diff
@@ -0,0 +1,10 @@
+
+BASE_SPACEX_URL = 'https://api.spacexdata.com/v3/'
+
+endpoints = {
+    'history': 'history',
+    'launches': 'launches',
+    'ships': 'ships',
+    'rockets': 'rockets'
+}
```

Comment:
dictionary usually is much more memory intensive than other data structure, in this particular case, since key and value are exactly the same, maybe we can further simplify this?